downstream/modules/platform/con-about-eda-controller.adoc

```diff
@@ -0,0 +1,11 @@
+[id="about-event-driven-ansible-controller_{context}"]
+
+= Event-Driven Ansible controller
+
+[role="_abstract"]
+The Event-Driven Ansible controller is the interface for event-driven automation and introduces automated resolution of IT requests. This component helps you connect to sources of events and acts on those events using rulebooks. This technology improves IT speed and agility, and enables consistency and resilience. With Event-Driven Ansible, you can: 
+
+* Automate decision making
+* Leverage numerous event sources
+* Implement event-driven automation within and across multiple IT use cases
+
```

Comment:
Not for this PR but I think it might be good to add a cross reference to the EDA doc once published.

Reply:
Hey @dcdacosta . I added the Additional Resources section and a temp (fake) link to the Getting Started w/ EDA guide. Let me know your thoughts.